### DISCUSSION TITLE
Idea: leverage go 1.20+ support for profile guided optimization

### DISCUSSION THREAD

**Discussion by dropwhile** [slackhq/nebula#1009] [Ideas]
from: https://go.dev/doc/pgo

> The standard approach to building is to store a pprof CPU profile with filename default.pgo in the main package directory of the profiled binary. By default, go build will detect default.pgo files automatically and enable PGO.

**Comment by dropwhile** [comment]
@nbrownus Was there a commit I missed where pgo was added?

**Reply by nbrownus** [reply]
There was not but I have opened a ticket in regards to it https://github.com/slackhq/nebula/issues/1042
